requirements.txt

```diff
@@ -1,3 +1,4 @@
-pygame>=2.1.0
+pygame~=2.6.0
 grpcio==1.59.3
-protobuf==4.23.2
\ No newline at end of file
+protobuf==4.23.2
+contextvars==2.4
```

Comment:
why contextvars? isn't that a [builtin](https://docs.python.org/3/library/contextvars.html)?

Reply:
seems to be related to https://youtrack.jetbrains.com/issue/PY-52554/Package-containing-module-contextvars-is-not-listed-in-the-project-requirements

Reply:
Yes, it is related to this issue. Should I remove this?